src/components/breadcrumbs.vue

```diff
@@ -13,12 +13,16 @@ except according to the terms contained in the LICENSE file.
   <div class="breadcrumbs">
     <template v-for="(link, index) in links" :key="index">
       <div class="breadcrumb-item" v-tooltip.text>
-        <router-link :to="link.path">
+        <router-link v-if="link.path != null" :to="link.path">
           <span v-if="link.icon" :class="link.icon"></span>
           {{ link.text }}
         </router-link>
+        <template v-else>
+          <span v-if="link.icon" :class="link.icon"></span>
+          {{ link.text }}
+        </template>
```

Comment:
This pattern has come up before of sometimes wrapping content in a link and sometimes not. We have the `Linkable` component that you could use for it. It makes it possible to avoid duplicating the inner content:

```suggestion
        <linkable :to="link.path">
          <span v-if="link.icon" :class="link.icon"></span>
          {{ link.text }}
        </linkable>
```